includes/definitions/hpe-mapdu.yaml

```diff
@@ -0,0 +1,26 @@
+os: hpe-mapdu
+text: 'HPE Managed Power Distribution Unit'
+type: power
+icon: hpe
+group: hpe
+over:
+    - {graph: device_power, text: Power}  
+    - {graph: device_current, text: Current}
+mib_dir: hp
+discovery:
+    -
+        sysObjectID:
+            - .1.3.6.1.4.1.232.165.7
+            - .1.3.6.1.4.1.232.165.11
+snmp_max_oid: '8'
+poller_modules:
+    wireless: false
```

Comment:
most of these modules are disabled by default, I prefer to leave it up to the user to set modules enabled/disabled.